.travis.yml

```diff
@@ -28,9 +28,7 @@ matrix:
 
 before_install:
     - composer self-update
-    - if [ "$SYMFONY_VERSION" != "" ]; then
```

Comment:
Crappy Insight :stuck_out_tongue: Happy to merge as I don't care, that's readable even on 1 line.